commands/stats.ts

```diff
@@ -0,0 +1,41 @@
+import { Message } from "discord.js";
+import { Command, CommandDefinition, Action } from ".";
+
+export const description: CommandDefinition = {
+  name: "Stat Hoarder",
+  description: "Displays server statistics",
+  usage: ["!stats"],
+  keys: ["stats"],
+};
+
+export const action = (message : Message, key : string) => {
+    const totalMembers= message.guild?.members.cache.size;  
+    let botCount = message.guild?.members.cache.filter(m => m.user.bot).size; 
+    if (botCount !== undefined) botCount++; // Bot doesn't include itself in the botCount lol
```

Comment:
Will this break if the command is called in a direct message?

Reply:
In a DM the bot will send something like this: 
![image](https://user-images.githubusercontent.com/43019056/104817122-4e73a700-57ed-11eb-92fa-2a79af46981c.png)

No errors are logged in the console, but if you'd like we can add a check to catch this case